lib/hydra/works/models/concerns/generic_work_behavior.rb

```diff
@@ -20,11 +20,48 @@ module GenericWorkBehavior
       type [RDFVocabularies::PCDMTerms.Object,WorksVocabularies::WorksTerms.GenericWork]
       include Hydra::Works::AggregatesGenericFiles
       include Hydra::Works::AggregatesGenericWorks
+      include Hydra::Works::BlockChildObjects
+
+      filters_association :members, as: :child_generic_works, condition: :works_generic_work?
```

Comment:
While a `PCDM::Work` could be a parent, I don't think a GenericWork could ever be a parent of a GenericWork, could it?